tflite/tflite-rtsp/demos/object-detection/object-detection.py

```diff
@@ -14,11 +14,17 @@
 
 
 ## Read Environment Variables
-USE_HW_ACCELERATED_INFERENCE = os.environ.get("USE_HW_ACCELERATED_INFERENCE")
+USE_HW_ACCELERATED_INFERENCE =True
+
+## The system returns the variables as Strings, so it's necessary to convert them where we need the numeric value
+if os.environ.get("USE_HW_ACCELERATED_INFERENCE") == "0":
+    USE_HW_ACCELERATED_INFERENCE = False
 
 MINIMUM_SCORE = os.environ.get("MINIMUM_SCORE")
 if not MINIMUM_SCORE:
     MINIMUM_SCORE = 0.55
+else:
```

Comment:
This cam be simplified to `MINIMUM_SCORE = float(os.environ.get("MINIMUM_SCORE", default=0.55))`.

Reply:
Same in line 41.